wpimath/src/main/java/edu/wpi/first/math/spline/QuinticHermiteSpline.java

```diff
@@ -63,9 +63,9 @@ public QuinticHermiteSpline(
   }
 
   /**
-   * Returns the coefficients matrix.
+   * Returns the matrix containing the coefficients.
```

Comment:
Given that coefficient matrix is the mathematical term, I'm not sure these changes should be made  https://en.wikipedia.org/wiki/Coefficient_matrix